plugins/contrast/index.js

```diff
@@ -6,6 +6,7 @@
 let $ = require("jquery");
 let Plugin = require("../base");
 let annotate = require("../shared/annotate")("labels");
+let blinder = require("../../node_modules/color-blind/lib/blind").Blind;
```

Comment:
You just call `require("color-blind").Blind` here and webpack will know where to look :)